keylime/tenant.py

```diff
@@ -156,6 +160,19 @@ def __init__(self) -> None:
     def verifier_base_url(self) -> str:
         return f"{bracketize_ipv6(self.verifier_ip)}:{self.verifier_port}"
 
+    def set_full_id_log_msg(self) -> None:
+        self.agent_fid_str = f"Agent {self.agent_uuid}"
```

Comment:
Curious, what does "fid" stand for?

Reply:
"full id"

Reply:
This is probably too nit picky, but in general with things like this I like naming them for what they will be used for, not what they contain. Something like "agent_log_str", etc might work better in case we ever decide what information should be in every log message (add to it, etc).